hs_core/tests/api/rest/base.py

```diff
@@ -0,0 +1,105 @@
+import socket
+
+from django.contrib.auth.models import Group
+
+from rest_framework.test import APIClient
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from hs_core.hydroshare import users
+from hs_core.hydroshare import resource
+
+
+class HSRESTTestCase(APITestCase):
+
+    def setUp(self):
+        self.hostname = socket.gethostname()
+        self.resource_url = "http://example.com/resource/{res_id}/"
+        self.maxDiff = None
+        self.client = APIClient()
+
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        # create a user
+        self.user = users.create_account(
+            'test_user@email.com',
+            username='testuser',
+            first_name='some_first_name',
+            last_name='some_last_name',
+            superuser=False)
+
+        self.client.force_authenticate(user=self.user)
+
+        self.resources_to_delete = []
+
+    def tearDown(self):
+        for r in self.resources_to_delete:
+            resource.delete_resource(r)
+
+        self.user.delete()
+
+    def getResourceBag(self, res_id):
+        """
+        Get resource bag from iRODS, following redirects
+        :param res_id: ID of resource whose bag should be fetched
+        :return: Django test client response object
+        """
```

Comment:
I'd try to stick with PEP-8 on docstrings: https://www.python.org/dev/peps/pep-0008/#documentation-strings

specifically the first line should be on its own with an empty line below.